modules/runner/src/integrationTest/java/org/apache/ignite/internal/jdbc/ItJdbcTest.java

```diff
@@ -98,7 +97,7 @@ void tearDown() {
         @DisplayName("Jdbc client can not connect without basic authentication configured")
         void jdbcCanNotConnectWithoutBasicAuthentication() {
             var url = "jdbc:ignite:thin://127.0.0.1:10800";
-            assertThrows(SQLException.class, () -> DriverManager.getConnection(url));
+            JdbcTestUtils.assertThrowsSqlException(SQLException.class, () -> DriverManager.getConnection(url));
```

Comment:
assertThrows**SqlException**(**SQLException**.class sounds strange

Reply:
yes! good catch. Seems I fix it already for anoter PR, but will fix here too